To-DoList/app/src/main/java/com/ezequielc/to_dolist/Singleton.java

```diff
@@ -0,0 +1,33 @@
+package com.ezequielc.to_dolist;
+
+import java.util.ArrayList;
+
+/**
+ * Created by student on 11/13/16.
+ */
+public class Singleton {
+    private static Singleton singleton = null;
+    private static ArrayList<Day> days;
+
+    public static Singleton getInstance() {
+        if (singleton == null) {
+            singleton = new Singleton();
+        } return singleton;
+    }
+
+    private Singleton() {
+        days = new ArrayList<>();
+    }
+
+    public ArrayList<Day> getDays() {
+        return days;
+    }
+
+    public void addDay(Day day){
+        days.add(day);
+    }
+
+    public void removeDay(Day day){
```

Comment:
It is better to remove the day by index rather than object reference. Although this works, it is not the preferred way.